docs/_source/conf.py

```diff
@@ -204,5 +205,11 @@
     ('py:class', 'sceptre.config.reader.Attributes'),
     ('py:class', 'sceptre.diffing.stack_differ.DiffType'),
     ('py:obj', 'sceptre.diffing.stack_differ.DiffType'),
-    ('py:class', 'TextIO')
+    ('py:class', 'DiffType'),
+    ('py:class', 'TextIO'),
+    ('py:class', '_io.StringIO'),
+    ('py:class', 'yaml.loader.SafeLoader'),
+    ('py:class', 'yaml.dumper.Dumper')
```

Comment:
I'd always end the line with `,` in Python. That way when someone adds another line later in this list it doesn't show up as a 2 line change. For example the TextIO line above would have been unmodified if it already had the trailing comma.

Reply:
That can be done.